lib/jellyfish/application.ex

```diff
@@ -7,18 +7,19 @@ defmodule Jellyfish.Application do
 
   @impl true
   def start(_type, _args) do
+    scrape_interval = Application.fetch_env!(:jellyfish, :metrics_scrape_interval)
```

Comment:
You can't call `compile_env` inside a function, so I would have to create a module attribute with the value.
I want to avoid, as I had problems with fetching compile-time variables in the past.